docs/content/en/tasks/scale/_index.md

```diff
@@ -188,7 +188,49 @@ The StatefulSet controller manages the deletion of Cassandra pods. It deletes on
 This means for example that `my-k8ssandra-dc1-default-sts-3` will be deleted before `my-k8ssandra-dc1-default-sts-2`.
 {{% /alert %}}
 
+## Bootstrap and decommission order
+
+A K8ssandra cluster will always bootstrap new nodes and decommission existing nodes in a strictly
+deterministic order that strives to achieve balanced racks.
+
+When scaling up, the operator will add new nodes to the racks with the fewest nodes. When scaling
+down, the operator will remove nodes from the racks with the most nodes. If two racks have the same
+number of nodes, the operator will start with the rack that comes first in the datacenter
+definition.
+
+Let's take a look at an example. Suppose we have a 3-node datacenter with the following racks:
+
+* `rack1` with 1 node
+* `rack2` with 1 node
+* `rack3` with 1 node
+
+Now let's scale up to, say, 8 nodes. The operator will bootstrap 5 new nodes, exactly in the
```

Comment:
cass-operator generates some k8s events when a node is added. It would be helpful to point that out. It would also be helpful to point out the status conditions that get set.

Reply:
The conditions ScalingUp and ScalingDown are already mentioned. I can mention the events too, but please keep in mind that in my experience, events are not reliable (some events don't show up in `kubectl get events`).

Reply:
I added a note about events, but again, as I was testing my own docs, I observed that some events get lost. In particular, I'm failing to see the ScalingDownRack events, even if I see them in the operator logs.